test/ModuleInterface/inlinable-function.swift

```diff
@@ -26,7 +26,6 @@ public struct Foo: Hashable {
 
   // CHECK: public var hasDidSet: [[INT]] {
   public var hasDidSet: Int {
-    // CHECK-NEXT: @_transparent get{{$}}
```

Comment:
Um. I don't think we want this to disappear, though, do we? Do we even support parsing a property with a set but no get?